packages/blade/src/_helpers/types.ts

```diff
@@ -1,13 +1,11 @@
-import type { ColorContrast } from '../tokens/theme/theme.d';
-
 /**
  * @template TokenType token type generic
  * @description Tokenises objects to dot notation strings, eg: `surface.text.normal.lowContrast`
  */
 type DotNotationColorStringToken<TokenType> = {
   [K in keyof TokenType]: `${Extract<K, number | string>}.${TokenType[K] extends Record<
     string,
-    ColorContrast | string
```

Comment:
Don't need this anymore since we have separate types for `ActionVariantsWithContrast` & `ActionVariants`